update-flake.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+# Copyright 2022 Tailscale Inc & Contributors
```

Comment:
2023

Reply:
No date at all

Reply:
yes, but we haven't updated this repo yet. I thought it would be better to be consistent and then clean up in a follow up.

Reply:
(I don't feel strongly one way or another, whatever you recommend @willnorris)